net-analyzer/gsad/files/gsad-22.init

```diff
@@ -3,20 +3,29 @@
 # Distributed under the terms of the GNU General Public License v2
```

Comment:
This file is referenced by two ebuilds after this change: gsad-22.5.1-r1 and gsad-22.6.0. But we are modifying this file here, which means the installed image of gsad-22.5.1-r1 changes, which requires a revision bump.

However, an alternative would be to drop gsad-22.5.1-r1 with this. I think we should do that.